content/docs/buildpacks/configuration.md

```diff
@@ -0,0 +1,258 @@
+*  [Cloud Native Buildpacks Bindings Specification][cnb bindings]. The original buildpacks bindings specification; this specification is en route to [deprecation][cnb bindings deprecation]
+
+Paketo Buildpacks will look for bindings in the `/platform/bindings` directory at build-time and in `$SERVICE_BINDING_ROOT` or `$CNB_BINDINGS` directory at runtime.
+
+For example, the Java Buildpack accepts a binding with `type` equal to `maven` containing a key named `settings.xml` containing [Maven settings][maven settings]. In the build container, the Maven Buildpack will use `settings.xml` if it finds either
+{{< code/copyable >}}
+/platform
+└── bindings
+    └── <name>
+        ├── settings.xml
+        └── type
+{{< /code/copyable >}}
+or
+{{< code/copyable >}}
+/platform
+└── bindings
+    └──<name>
+        └── metadata
+        |   └── kind
+        └── secret
+            └── settings.xml
+{{< /code/copyable >}}
+on the filesystem, where either the `type` or `kind` file contains the string `maven`.
+
+
+#### How to use bindings?
+The workflow for creating a binding and providing it to a build will depend on the chosen platform. For example, `pack` users should use the `--volume` flag to mount a binding directory into the build or app containers. Users of the `kpack` platform should store key value pairs in a Kubernetes Secret and provide that secret and associated metadata to an Image as described in the [kpack documentation][kpack service bindings].
+
+
+**Example:**  Providing a Binding to `pack build`
+
+Given a directory containing a build-time binding, `pack` users can provide this binding to a Paketo buildpack using the `--volume` flag.
+{{< code/copyable >}}
+pack build --volume <absolute-path-to-binding>:/platform/bindings/<binding-name> <image-name>
+{{< /code/copyable >}}
+
+**Example:**  Providing a Binding to `docker run`
+
+Given a directory containing a runtime binding, `docker` users can provide the binding to the app image using the `--volume` and `--env` flags
+{{< code/copyable >}}
+pack build --env SERVICE_BINDING_ROOT=/bindings --volume <absolute-path-to-binding>:/bindings/<binding-name> <image-name>
+{{< /code/copyable >}}
+
+### Procfiles
+Paketo users may override buildpack-provided types or augment the app-image with additional process types using a `Procfile`.  `Procfile` support is provided by the [Paketo Procfile Buildpack][bp/procfile]. The Procfile Buildpack will search for a file named `Procfile` at the root of the application. `Procfiles` should adhere to the following schema:
+```
+<type>: <command>
+```
+
+If a given [language family buildpack][language family buildpacks] does not contain the Procfile Buildpack it can be explicitly appended at runtime.
+
+**Example**: A Hello World Procfile
+
+The following adds a process with `type` equal to `hello` and makes it the default process.
+{{< code/copyable >}}
+echo "hello: echo hello world" > nodejs/yarn
+pack build samples/nodejs \
+  --path nodejs/yarn \
```

Comment:
```suggestion
  --path nodejs/yarn/Procfile \
```

Reply:
Did you mean this line instead? https://github.com/paketo-buildpacks/paketo-website/pull/48/commits/301240792c6cd0b99fb4ad76c2a91ae22a6b1cf7#diff-011af7d1796feaa60e53963bb34a6c24R179

Reply:
Oops. Yeah. Your latest change looks good to me.